src/manifold.cpp

```diff
@@ -382,17 +386,10 @@ double Manifold::GetEpsilon() const {
   return GetCsgLeafNode().GetImpl()->epsilon_;
 }
 
-Manifold Manifold::SetEpsilon(double epsilon) const {
-  auto impl = std::make_shared<Impl>(*GetCsgLeafNode().GetImpl());
-  auto oldTolerance = impl->tolerance_;
-  impl->SetEpsilon(epsilon);
-  if (impl->tolerance_ > oldTolerance) impl->SimplifyTopology();
-  return Manifold(impl);
-}
-
 /**
- * Returns the tolerance of this Manifold's vertices.
- * Edges shorter than this tolerance value will be collapsed.
+ * Returns the tolerance value of this Manifold. Triangles that are coplanar
+ * within tolerance tend to be merged and edges shorter than tolerance tend to
```

Comment:
I'm thinking if this will be true later when we implement better simplification. Not sure how to express this properly, but the final result should be between dilation and erosion by epsilon?